apps/export/service.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import json
+from superdesk.services import BaseService
+from superdesk import get_resource_service
+from superdesk.errors import SuperdeskApiError
+from superdesk.publish.formatters import get_all_formatters
+from eve.validation import ValidationError
+from io import BytesIO
+from zipfile import ZipFile
+from flask import current_app as app
+
+logger = logging.getLogger(__name__)
+
+FORMATTER_FORMAT_TYPES = {'AAPIpNewsFormatter': 'AAP IPNEWS', 'NINJSFormatter': 'ninjs'}
```

Comment:
Tks Mayur. I have added these attributes(format_type and output_field) to these specific formatters directly.